controller_manager:
  ros__parameters:
    update_rate: 80  # Hz

    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster

    leader_follower_controller:
      type: koch_controllers/LeaderFollowerController

leader_follower_controller:
  ros__parameters:
    leader_joints:
      - leader_joint1
      - leader_joint2
      - leader_joint3
      - leader_joint4
      - leader_joint5
      - leader_joint_gripper
    follower_joints:
      - follower_joint1
      - follower_joint2
      - follower_joint3
      - follower_joint4
      - follower_joint5
      - follower_joint_gripper

    leader_to_follower_offset:
      - 3.14159
      - 0.
      - 0.
      - 0.
      - 3.14159
      - 0.

    leader_to_follower_scale:
      - 1.
      - 1.
      - -1.
      - 1.
      - 1.
      - 1.

    command_interfaces:
      - position

    state_interfaces:
      - position
